### DISCUSSION TITLE
Bulk expiration of entries / Expired Groups

### DISCUSSION THREAD

**Discussion by sovamind** [keepassxreboot/keepassxc#7975] [Ideas]
Right now you can expire a group and it does not expire all the entries inside it.  When KeepassXC is integrated with a browser, all the entries in the expired group are returned.  It seems like it would be better if:

- When you expire a group, no entries inside it should be returned to the browser extensions
OR
- When you expire a group, all entries inside the group should have their expiration set to the new expiration date/time for the group

It also might be nice to have a way to bulk expire entries by selecting multiple entries, then go to the entries menu, bulk expire.  Or even to offer a "bulk edit" option, so that you could update multiple entries to all have the same values, but leaving differences alone, such as needing to update a favicon or tags.

**Comment by droidmonkey** [comment]
You can expire a group? Also, you can exclude groups and their entries from the browser extension in the browser settings tab of the group (version 2.7.0+)

**Comment by sovamind** [comment]
Yes, you can mark a group as expired. Right click, choose edit group, then check expiration date and enter a time/date.  However, it doesn't seem to do much in KeepassXC.  I don't want to exclude groups unless they are expired.  Basically, I have groups for clients, and when I'm no longer working with them, I want to expire the group and then have their logins no longer suggested.
________________________________

**Reply by droidmonkey** [reply]
The equivalent for now is to exclude the expired group from browser in the browser settings tab.
